lib/hal/hal_ssd1608.ex

```diff
@@ -71,14 +68,15 @@ defmodule Inky.HAL.PhatSSD1608 do
 
   @impl Inky.HAL
   def handle_update(pixels, border, push_policy, state = %State{}) do
-    black_bits = PixelUtil.pixels_to_bits(pixels, @rows, @cols, @rotation, @color_map_black)
-    accent_bits = PixelUtil.pixels_to_bits(pixels, @rows, @cols, @rotation, @color_map_accent)
+    %{width: width, height: height, rotation: rotation} = state.display
+    black_bits = PixelUtil.pixels_to_bits(pixels, width, height, rotation, @color_map_black)
+    accent_bits = PixelUtil.pixels_to_bits(pixels, width, height, rotation, @color_map_accent)
```

Comment:
Why did rows become width here? This means that the axes have been swapped, was that done on purpose? This will break code that is built on the current dimensions.